ddp-workspace/projects/ddp-singular/src/app/components/footer/footer.component.html

```diff
@@ -62,10 +62,10 @@
             </a>
           </li>
           <li class="link-list__item link-list__item_highlight-stroke_invert" [routerLinkActiveOptions]="{exact: true}" routerLinkActive="link-list__item_active">
-            <ddp-sign-in-out [signOutReturnToUrl]="Route.Home"></ddp-sign-in-out>
+            <ddp-sign-in-out (signIn)="signIn()" [signOutReturnToUrl]="Route.Home"></ddp-sign-in-out>
           </li>
           <li class="link-list__item link-list__item_highlight link-list__item_highlight_invert" [routerLinkActiveOptions]="{exact: true}" routerLinkActive="link-list__item_active" *ngIf="!isAuthenticated">
-            <a class="button button--white" [routerLink]="Route.PreScreening">
+            <a (click)="signUp($event)" class="button button--white" [routerLink]="Route.PreScreening">
```

Comment:
Actually, the method `signUp` does not do a sign up but emit a sign-up analytics event.
May be it is worth renaming it (`signIn` as well, line 65)